src/app.js

```diff
@@ -168,9 +168,6 @@ class SAFEApp extends EventEmitter {
     if (!url) return Promise.reject(makeError(errConst.MISSING_URL.code, errConst.MISSING_URL.msg));
 
     const parsedUrl = parseUrl(url);
-    if (!parsedUrl) {
```

Comment:
Why this is not needed anymore?
Also, we need to update the documentation as it seems it now accepts either an object or an array.

Reply:
This line isn't needed anymore because `!url` is already handled on line 168. If there is an error parsing the url it will be handled by the the `parseUrl` function.

Reply:
What is it that accepts an object or an array? The `webFetch` function for it's `url` parameter? I just see that we handle objects for `options` parameter.

Reply:
Currently `Promise.reject(makeError(errConst.MISSING_URL.code, errConst.MISSING_URL.msg));` is returned if `!parseUrl`, but this condition can never be met because Node's [url.parse](https://nodejs.org/docs/latest-v8.x/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost) function will throw an error if no URL string is passed to it and also the function doesn't consider any string to be an invalid URL, it will simply return an [URL](https://nodejs.org/docs/latest-v8.x/api/url.html#url_class_url) object.